catalog/core/catalog-core-camelcomponent/src/main/java/ddf/camel/component/catalog/content/ContentProducer.java

```diff
@@ -121,13 +151,25 @@ public void process(Exchange exchange)
 
         if (StringUtils.isNotEmpty(mimeType)) {
             ContentItem newItem;
-            newItem = new ContentItemImpl(Files.asByteSource(ingestedFile), mimeType,
-                    ingestedFile.getName(), null);
+            newItem = new ContentItemImpl(Files.asByteSource(ingestedFile),
+                    mimeType,
+                    ingestedFile.getName(),
+                    null);
 
             LOGGER.debug("Creating content item.");
 
-            CreateStorageRequest createRequest = new CreateStorageRequestImpl(
-                    Collections.singletonList(newItem), null);
+            Map<String, Serializable> propertiesList = new HashMap<>();
+
+            headerMap.entrySet()
+                    .forEach((entry) -> {
+                        if (!CAMEL_DEFAULT_HEADERS.contains(entry.getKey())) {
```

Comment:
Instead of adding any property except for the camel ones, why not only choose a particular property from the camel route?  That seems safer to me.